frontend/components/Form.js

```diff
@@ -152,14 +160,24 @@ class Form extends Component {
       help,
     } = field
 
+    const {
+      [`field-${name}`]: value,
+      [`editorState-${name}`]: editorState,
+    } = this.state
+
+    const isHorizontal =
+      typeof this.props.isHorizontal !== 'undefined'
+        ? this.props.isHorizontal
+        : true
```

Comment:
Why can't we just have like
```js
const isHorizontal = Boolean(this.props.isHorizontal)
```

Reply:
actually why don't we just use the prop itself? Seems like it should be a boolean

Reply:
I tried destructuring the prop and the suggestion above, both break the form for some reason

Reply:
fixed, used default arg in destructure to get the same behavior